xbmc/dbwrappers/mysqldataset.cpp

```diff
@@ -158,9 +168,6 @@ int MysqlDatabase::connect(bool create_new) {
       }
       else if (create_new)
       {
-        char sqlcmd[512];
-        int ret;
```

Comment:
I've seen that, just checked the implementation for ```query_with_reconnect()``` and it will overwrite ```ret``` with 0 when it internally fails. So, we're good.